### PR TITLE
Feature/upgrade to 14.1

### PR DESCRIPTION
**Pull request by jared-improbable**

### CODE REVIEW

File: Workers/DatabaseSyncWorker/DatabaseChanges.cs [hunk 1]
```diff
@@ -60,7 +69,10 @@ public DatabaseChanges(PostgresOptions postgresOptions, string tableName)
                     }
                     catch (Exception e)
                     {
-                        Log.Error(e, "LISTEN {TableName}", tableName);
+                        if (!tcs.Token.IsCancellationRequested)
```
Comment (jamiebrynes7) [inline Workers/DatabaseSyncWorker/DatabaseChanges.cs:72]: Is this to filter out cancelled exceptions? If so you could pattern match on the type of `e` here

File: scripts/BuildNugetPackages/Program.cs [hunk 2]
```diff
@@ -116,22 +116,22 @@ private static void BuildPackages(string nugetSourceDir)
...
+                Path.GetFullPath(Path.Combine(nugetSourceDir, "Improbable", "WorkerSdkInterop", "Improbable.WorkerSdkInterop"));
             var localNugetPackages = Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, "nupkgs"));
 
             // For simplicity, some packages depend on Improbable.WorkerSdkInterop. Make sure that's packaged first in the source directory.
             var targetPath = Path.GetFullPath(Path.Combine(nugetSourceDir, "nupkgs"));
             CleanDirectory(targetPath);
 
-            shell.Run("dotnet", "pack", $"\"{sdkInteropDir}\"", "--verbosity:quiet", "-p:Platform=x64", "--output",
-                $"\"{targetPath}\"")
+            shell.Run("dotnet", "pack", $"{sdkInteropDir}", "--verbosity:quiet", "-p:Platform=x64", "--output",
```
Comment (jamiebrynes7) [inline scripts/BuildNugetPackages/Program.cs]:
```suggestion
            shell.Run("dotnet", "pack", sdkInteropDir, "--verbosity:quiet", "-p:Platform=x64", "--output",
```

File: scripts/BuildNugetPackages/Program.cs [hunk 3]
```diff
@@ -116,22 +116,22 @@ private static void BuildPackages(string nugetSourceDir)
...
+                $"{targetPath}")
                 .RedirectTo(Console.Out)
                 .RedirectStandardErrorTo(Console.Error)
                 .Wait();
 
             // Now build everything into the worker's directory.
-            shell.Run("dotnet", "pack", $"\"{Path.Combine(nugetSourceDir, "Improbable")}\"", "--verbosity:quiet",
-                "-p:Platform=x64", "--output", $"\"{localNugetPackages}\"")
+            shell.Run("dotnet", "pack", $"{Path.Combine(nugetSourceDir, "Improbable")}", "--verbosity:quiet",
```
Comment (jamiebrynes7) [inline scripts/BuildNugetPackages/Program.cs]:
```suggestion
            shell.Run("dotnet", "pack", Path.Combine(nugetSourceDir, "Improbable"), "--verbosity:quiet",
```

File: scripts/BuildNugetPackages/Program.cs [hunk 4]
```diff
@@ -116,22 +116,22 @@ private static void BuildPackages(string nugetSourceDir)
...
+                "-p:Platform=x64", "--output", $"{localNugetPackages}")
```
Comment (jamiebrynes7) [inline scripts/BuildNugetPackages/Program.cs]:
```suggestion
                "-p:Platform=x64", "--output", localNugetPackages)
```

**Review by jamiebrynes7 — APPROVED** [review]

File: Workers/DatabaseSyncWorker/DatabaseChanges.cs (hunk 1 above)
Comment (jared-improbable) [reply Workers/DatabaseSyncWorker/DatabaseChanges.cs:72]: This is to filter out exceptions that are thrown as a result of cancelling, the catch block above should catch the explicit `TaskCanceledException` type

**Pull request merged**